tools/topology/topology1/m4/multiband_drc_coef_passthrough.m4

```diff
@@ -0,0 +1,70 @@
+# Exported Control Bytes 16-Aug-2023
+CONTROLBYTES_PRIV(MULTIBAND_DRC_priv,
```

Comment:
This is topology1 seperate PR ?

Reply:
I've split the tplg1 blobs update to #8079. Next update to this PR won't contain those.